Sources/PKYouboraAdsAdapter.swift

```diff
@@ -198,6 +234,7 @@ extension PKYouboraAdsAdapter {
                 messageBus.addObserver(self, events: [e.self]) { [weak self] event in
                     guard let self = self else { return }
                     self.fireStop(["skipped":"true"])
```

Comment:
I know JS changed this to `fireSkip`, is this also same for iOS?

Reply:
you are right, just changed this for `fireSkip`